nodeJS/APIs/APIs.md

```diff
@@ -12,41 +12,42 @@ By the end of this lesson, you should be able to do the following:
 - Explain CORS headers
 
 ### Introduction
-In recent years, a new pattern for developing websites has been gaining popularity. Instead of creating an app that hosts both the database and view templates, many developers are separating these concerns into separate projects, hosting their database on a server (either on something like [Heroku](https://www.heroku.com/) or on a VPS like [Digital Ocean](https://www.digitalocean.com/)), then using a service such as [GitHub Pages](https://pages.github.com/) or [Netlify](https://www.netlify.com/) to host their frontend.
+In recent years, a new pattern for developing websites has been gaining popularity. Instead of creating an app that hosts both the database and view templates, many developers are separating these concerns into separate projects, hosting their backend and database on a server (either on something like [Heroku](https://www.heroku.com/) or on a VPS like [Digital Ocean](https://www.digitalocean.com/)), then using a service such as [GitHub Pages](https://pages.github.com/) or [Netlify](https://www.netlify.com/) to host their frontend. This technique is sometimes referred to as the [Jamstack](https://jamstack.org/what-is-jamstack/).
 
-Organizing your project this way can be beneficial because it allows your project to be more modular. This also allows you to use a single back-end source for multiple front-end applications, such as a website, a desktop app, or a mobile app. Other developers enjoy this pattern because they simply like using front-end frameworks such as React or Vue to create nice front-end-only, single-page applications.
+Organizing your project this way can be beneficial because it allows your project to be more modular instead of combining business logic with rendering frontend rendering.  This also allows you to use a single backend source for multiple frontend applications, such as a website, a desktop app, or a mobile app.
```

Comment:
> rendering frontend rendering.

I would just say "combining business logic with view logic." Right now, the end of the sentence reads a little funny.

Reply:
ha that is definitely terrible wording on my part. fixed with your suggestion.

>it only mentions that there are developers who prefer to work in the frontend alone which is valuable, anecdotal context.

So I quoted the preceding sentence below. I guess I take issue with the paragraph making it seem necessary to use GH pages / Netlify in order to leverage React / Vue or to achieve modularity. Added it back to keep this moving and I'm sure there's better ways to demonstrate my line of thought.

>Instead of creating an app that hosts both the database and view templates, many developers are separating these concerns into separate projects, hosting their backend and database on a server (either on something like [Heroku](https://www.heroku.com/) or on a VPS like [Digital Ocean](https://www.digitalocean.com/)), then using a service such as [GitHub Pages](https://pages.github.com/) or [Netlify](https://www.netlify.com/) to host their frontend.